.flake8

```diff
@@ -1,7 +1 @@
-[flake8]
-inline-quotes = "
-ignore = E203, E266, W503, ANN002, ANN003, ANN101, ANN102, ANN401, N807, N818
-max-line-length = 79
-max-complexity = 18
-select = B,C,E,F,W,T4,B9,ANN,Q0,N8,VNE
-exclude = venv, tests
+[flake8]
inline-quotes = "
ignore = E203, E266, W503, ANN002, ANN003, ANN101, ANN102, ANN401, N807, N818
max-line-length = 79
max-complexity = 18
select = B,C,E,F,W,T4,B9,ANN,Q0,N8,VNE
exclude = venv, tests

```

Comment:
I think you should close the PR, and reopen it with the new solution. Make sure flake8 and tests are passing

Reply:
If you need help, feel free to go to the Q&A